packages/inspire/InspireGateway.js

```diff
@@ -21,17 +21,18 @@ import EngineContentAPI from "~/engine/EngineContentAPI";
 import extendValaaSpaceWithEngine from "~/engine/ValaaSpace";
 
 import InspireView from "~/inspire/InspireView";
+import PerspireView from "~/inspire/PerspireView";
+
 import { registerVidgets } from "~/inspire/ui";
 import type { Revelation } from "~/inspire/Revelation";
 import extendValaaSpaceWithInspire from "~/inspire/ValaaSpace";
 
-import { getDatabaseAPI } from "~/tools/indexedDB/getRealDatabaseAPI";
 import { arrayBufferFromBase64 } from "~/tools/base64";
 import { invariantify, LogEventGenerator, valaaUUID } from "~/tools";
+import isInBrowser from "is-in-browser";
```

Comment:
Good find. The implementation for this is quite simple though:
https://github.com/tuxsudo/is-in-browser/blob/master/src/index.js
We probably should think about just fixing the ~/tools/wrapError.js:inBrowser to use same mechanism.
Also to extract the inBrowser function from wrapError to inBrowser.js.

There's a philosophical question on what to do with the @valos/tools library anyway.